tests/providers/google/cloud/hooks/test_cloud_run.py

```diff
@@ -294,3 +301,73 @@ def mock_get_operation(self, expected_operation):
 
     def _dummy_get_credentials(self):
         pass
+
+
+@pytest.mark.db_test
+class TestCloudRunServiceHook:
+    def dummy_get_credentials(self):
+        pass
+
+    @pytest.fixture
+    def cloud_run_service_hook(self):
+        cloud_run_service_hook = CloudRunServiceHook()
+        cloud_run_service_hook.get_credentials = self.dummy_get_credentials
+        return cloud_run_service_hook
+
+    @mock.patch(
+        "airflow.providers.google.common.hooks.base_google.GoogleBaseHook.__init__",
+        new=mock_base_gcp_hook_default_project_id,
+    )
+    @mock.patch("airflow.providers.google.cloud.hooks.cloud_run.ServicesClient")
+    def test_create_service(self, mock_batch_service_client, cloud_run_service_hook):
+        service_name = "service1"
+        region = "region1"
+        project_id = "projectid"
+        service = Service()
+
+        create_request = CreateServiceRequest()
+        create_request.service = service
+        create_request.service_id = service_name
+        create_request.parent = f"projects/{project_id}/locations/{region}"
+
+        cloud_run_service_hook.create_service(
+            service=Service.to_dict(service), service_name=service_name, region=region, project_id=project_id
+        )
+        cloud_run_service_hook._client.create_service.assert_called_once_with(create_request)
+
+
+class TestCloudRunServiceAsyncHook:
```

Comment:
The same here, we need a test for delete method

Reply:
done